dash_cytoscape/package.json

```diff
@@ -10,9 +10,10 @@
     "build:js": "webpack --mode production",
     "build:js-dev-extra": "webpack --config webpack.dev.extra.config.js",
     "build:js-extra": "webpack --config webpack.prod.extra.config.js",
+    "build:js-all": "npm run build:js && npm run build:js-dev && npm run build:js-dev-extra && npm run build:js-extra",
```

Comment:
Any particular reason that you changed this file?

Reply:
There's two new builds I need to run in order to generate the new dash_cytoscape_extra bundles.